404.md

```diff
@@ -83,28 +91,13 @@ if (gonnaFwd) {
   window.location.href = newURL;
   document.write('<meta http-equiv="refresh" content="0; url=' + newURL + '">')
 } else {
-  function populateTicket()
-  {
-    var searchVal = phrase;
-    var output = new Array();
-    output.push("<ul><li><a href='https://github.com/docker/docker.github.io/issues/new?title=404 at: ");
-    output.push(path);
-    output.push("&body=URL: ");
-    output.push(path);
-    output.push("' class='nomunge'>File a ticket</a></li>");
-    output.push("<li><a href='/search/?q=" + phrase + "'</a>Search for <tt>&quot;" + phrase + "&quot;</tt></a></li>");
-    output.push("<li><a href='/search/'>Try a different search</a></li>");
-    output.push("<li><a href='/'>Go to the index</a></li></ul><br />");
-    document.getElementById("sorryMsg").innerHTML = output.join("");
-  }
-  window.onload = populateTicket;
+  console.log("Showing 404 choices");
+  var searchVal = phrase;
+  var output = new Array();
+  output.push("<ul><li><a href='https://github.com/docker/docker.github.io/issues/new?title=404 at: " + path + "&body=URL: " + path + "' class='nomunge'>File a ticket</a></li>");
```

Comment:
This isn't new code, just migrated from elsewhere, and it WFM ... I did think maybe it needed `&nbsp` treatment, but it seems OK.